.gitignore

```diff
@@ -9,6 +9,20 @@
 
 # Where 3rd-party dependencies like ExDoc output generated docs.
 /doc
+/v0.1.0
+/v0.10.0
+/v0.9.0
+/v1.0.0
+/v1.0.3
+/v1.0.4
+/v1.0.5
+/v1.0.6
+/v2.0.0
+/v2.0.0-alpha.4
+/v2.0.0-beta.1
+/v2.0.1
+/v2.0.2
+/v2.1.0-alpha.1
```

Comment:
Can this be reduced to something around `/v*/`?

Reply:
👍